backend/src/email/routes/tests/email-transactional.routes.test.ts

```diff
@@ -142,92 +201,435 @@ describe('POST /transactional/email/send', () => {
     expect(mockSendEmail).not.toBeCalled()
   })
 
-  test('Should send a message with a valid attachment', async () => {
-    const mockSendEmail = jest
-      .spyOn(EmailService, 'sendEmail')
-      .mockResolvedValue('message_id')
+  test('Should throw an error if email subject or body is empty after removing invalid HTML tags', async () => {
```

Comment:
This test case is testing way more than this title suggests, it's testing for the email being persisted in the db with the corresponding error as well

Reply:
OK will fix the title